### DISCUSSION TITLE
Taking HFE copies of MFS Mac 400k disks

### DISCUSSION THREAD

**Discussion by cknowlesuk** [keirf/greaseweazle#511] [Q&A] [answered]
Am I correct in thinking that there is no way to take a HFE copy from a MFS formatted Mac 400k disk? I've tried variously with 250, 375, and 500 bitrate specified, but what I'm getting out has notably fewer tracks (when analysed in HxC) than if I take a SCP or RAW copy (without specifying the RPM at all); the latter look the same, and if I use HxC to convert eg the SCP copy to a HFE file, then it looks notably different to any of the HFE copies I was taking directly.

This all arising from a desire to have a simple process to image all disks initially, with a view to converting them to something more appropriate (specifying tracks, sides, etc) later, and hoping I could do this with HFE, as the single file output with a notably smaller filesize than SCP; while this works fine for MFM (where its basically just a case of identifying HD vs DD), I think the old Apple things might not be suitable for such a workflow?

**Comment by keirf** [comment, marked as answer]
Indeed that won't really work well, as the bitrate (or rpm) varies across tracks. You are better off imaging to SCP. If you are confident that your disks are plain 400k or 800k disks, then gw can even convert these directly:
```
gw read --format=mac.400 name.dsk
```
Or if you want to take a raw SCP dump and then convert:
```
gw read name.scp
gw convert --format=mac.400 name.scp name.dsk
```
Or
```
gw convert --format=mac.800 name.scp name.dsk
```
If you are confident that the disk is single sided, you can dump the single side only to SCP:
```
gw read --tracks=h=0 name.scp
```
